docs/data/data-grid/prompt/prompt.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Data Grid - Prompt
+---
+
+# Data Grid - Prompt [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+
+<p class="description">Translates natural language into a set of grid state updates and applies those to the Data Grid component.</p>
+
+:::warning
+To use this feature, you need to have a prompt processing backend. MUI offers this service as a part of a premium package add-on. Check [licensing page](/x/introduction/licensing/) for more information.
+:::
+
+The prompt feature allows users to interact with the Data Grid component using natural language. The user can type commands like "sort by name" or "show amounts larger than 1000" in the prompt input field, and the Data Grid will update accordingly.
+
+To increase the accuracy of the prompt processing, the user should provide example values for the available columns. This can be done in following ways.
```

Comment:
```suggestion
To increase the accuracy of the prompt processing, the user should provide example values for the available columns.
This can be done in following ways.
```